### ISSUE TITLE
Unable to produce accessibility score "?" running lighthouse under mobile flag

### ISSUE THREAD

**Issue by TimRoenicke** [dequelabs/axe-core#3062]
Product: axe-core

Expectation: Produce Accessibility score for mobile

Actual: Error! axe-core error cannot read property of 'cells' undefined - Works when running desktop but not mobile on [fullertontool.com](https://fullertontool.com)

<img width="784" alt="CleanShot 2021-07-06 at 13 45 12@2x" src="https://user-images.githubusercontent.com/2480028/124644732-94d2c600-de60-11eb-833c-bddf808d9b5a.png">

Motivation: 

<pre><code>
axe-core version: Chrome 91.0.4472.114

Browser and Assistive Technology versions

For Tooling issues:
- Platform:  Mac  Chrome 91.0.4472.114
</code></pre>

**Comment by straker** [comment]
Thanks for the issue. Based on the error I believe this might be related to https://github.com/dequelabs/axe-core/issues/2674, but will dig further at a later time to see if it is.
